RedHatInsights/insights-remediations-frontend
# fix(REMEDY-277): extract columns from reducer hash

# Description

Associated Jira ticket: [#REMEDY-277](https://issues.redhat.com/browse/REMEDY-277)

This extracts adding columns from reducers to resolve the error. Instead, columns will be provided by columns callback function.


# How to test the PR

1. Open a remediation detail page by clicking one of the remediation name link
2. visit the 'Activity' tab
3. Open the executer detail page by selecting connection name link
4. Observe that page loads as expected

# Before the change


# After the change


# Dependent work link


# Checklist:
